utils/renderEslint.ts

```diff
@@ -49,8 +49,8 @@ function configureEslint({ language, styleGuide, needsPrettier, needsCypress, ne
     const cypressOverrides = [
       {
         files: needsCypressCT
-          ? ['**/__tests__/*.spec.{js,ts,jsx,tsx}', 'cypress/integration/**.spec.{js,ts,jsx,tsx}']
-          : ['cypress/integration/**.spec.{js,ts,jsx,tsx}'],
+          ? ['**/__tests__/*.{cy,spec}.{js,ts,jsx,tsx}', 'cypress/e2e/**.{cy,spec}.{js,ts,jsx,tsx}']
+          : ['cypress/e2e/**.{cy,spec}.{js,ts,jsx,tsx}'],
```

Comment:
I really prefer not using `__test__` but it's a conversation topic for another time.

`.spec.` will conflict with Vitest types I believe, which is why we chose `.cy.` but it's okay, you're right about the struggle people may have with migration.

Reply:
They will not in our case because of the usage of solution-style tsconfigs.
But yeah, people who are not familiar enough with TypeScript will find this troublesome. So I've changed the default pattern.